frontend/src/pages/Dashboard/components/DayAgenda/DayAgenda.module.scss

```
@@ -152,6 +152,8 @@
   border-radius: 999px;
   background: var(--surface);
   color: var(--ink-2);
+  // <a> 와 <button> 두 모양으로 씁니다. 버튼은 글꼴을 물려받지 않아 따로 맞춥니다.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)